packages/documentsearch-extension/schema/plugin.json

```
@@ -47,7 +47,15 @@
       "selector": ".jp-mod-searchable"
     }
   ],
-  "properties": {},
+  "properties": {
+    "searchDebounceTime": {
+      "title": "Search debounce time (ms)",
+      "description": "The debounce time in milliseconds applied to the search input field. The already opened input fileds will not be updated if you change that value",
```

Small typo and we are good to go.

```suggestion
      "description": "The debounce time in milliseconds applied to the search input field. The already opened input files will not be updated if you change that value",
```